Libraries/Modal/Modal.js

```diff
@@ -58,6 +59,7 @@ class Modal extends React.Component {
     return (
       <RCTModalHostView
         animated={this.props.animated}
```

Comment:
Can we keep the backwards-compatible check for animated in the JS here. That way we don't to maintain it twice in native. If animationType is undefined and animated is set, set animationType to slide.